lncfg/db.go

```diff
@@ -562,6 +602,20 @@ func (db *DB) GetBackends(ctx context.Context, chanDBPath,
 	}
 	closeFuncs[NSMacaroonDB] = macaroonBackend.Close
 
+	peerStorageBackend, err := kvdb.GetBoltBackend(&kvdb.BoltBackendConfig{
+		DBPath:            chanDBPath,
```

Comment:
I used the `ChanDBPath` here, hope that is aligned

Reply:
Yeah I think that's a reasonable decision.